changestreams.go

```diff
@@ -17,6 +17,14 @@ const (
 	UpdateLookup = "updateLookup"
 )
 
+type ChangeDomainType int
```

Comment:
As this is an implementation detail (never exposed / used by an `mgo` user) we should probably not export it, or it's variants below.

Reply:
Ok that makes sense, i'll do it ASAP.

Thanks @domodwyer

Reply:
Fixed it @domodwyer .
As usual @eminano, travis randomly failed a build... :D